sw/device/tests/sim_dv/flash_ctrl_lc_rw_en_test.c

```diff
@@ -69,24 +69,25 @@ const bool kProdExpectedAccess[5] = {true, true, true, true, false};
 
 static bool access_partitions(bool do_write, bool do_read, int page_id,
                               const uint32_t *data, uint32_t size) {
-  uint32_t address = flash_ctrl_testutils_info_region_setup(
-      &flash, page_id, kBankId, kPartitionId);
+  uint32_t address;
+  CHECK_STATUS_OK(flash_ctrl_testutils_info_region_setup(
+      &flash, page_id, kBankId, kPartitionId, &address));
 
   bool retval = true;
```

Comment:
The whole code below is a bit strange. Is a there a purpose in not returning early on failure?

Reply:
I looked it again, and It seems there's no problem in returning earlier, but I prefer don't change the test in this PR.